app/views/miq_ae_customization/_old_dialogs_form.html.haml

```diff
@@ -1,33 +1,44 @@
 - url = url_for(:action => 'old_dialogs_form_field_changed', :id => "#{@dialog.id || 'new'}")
 #form_div
   = render :partial => "layouts/flash_msg"
-  %h3=_('Basic Information')
-  %table.style1
-    %tr
-      %td.key=_('Name')
-      %td.wide
+  %h3
+    = _('Basic Information')
+  %form.form-horizontal
+    .form-group
+      %label.col-md-2.control-label
+        = _('Name')
+      .col-md-8
         = text_field_tag("name",
-          @edit[:new][:name],
-          :maxlength         => MAX_NAME_LEN,
-          "data-miq_observe" => {:interval => '.5', :url => url}.to_json)
+                          @edit[:new][:name],
+                          :maxlength         => MAX_NAME_LEN,
+                          :class => "form-control",
+                          "data-miq_observe" => {:interval => '.5', :url => url}.to_json)
         = javascript_tag(javascript_focus('name'))
-    %tr
-      %td.key=_('Description')
-      %td.wide
+    .form-group
+      %label.col-md-2.control-label
+        = _('Description')
+      .col-md-8
         = text_field_tag("description",
-          @edit[:new][:description],
-          :maxlength         => 100,
-          "data-miq_observe" => {:interval => '.5', :url => url}.to_json)
-    %tr
-      %td.key=_('Type')
-      - dialog_types = @dialog.dialog_type ? MiqDialog::DIALOG_TYPES.sort :  [["<#{_('Choose')}>", nil]] + MiqDialog::DIALOG_TYPES.sort
-      %td.wide
+                          @edit[:new][:description],
+                          :maxlength         => 100,
+                          :class => "form-control",
+                          "data-miq_observe" => {:interval => '.5', :url => url}.to_json)
+    .form-group
+      %label.col-md-2.control-label
+        = _('Type')
+        - miq_dialog_s = MiqDialog::DIALOG_TYPES.sort
+        - dialog_types = @dialog.dialog_type ? miq_dialog_s : [["<#{_('Choose')}>", nil]] + miq_dialog_s
+      .col-md-8
         = select_tag('dialog_type',
-          options_for_select(dialog_types,
-          @edit[:new][:dialog_type]),
-          "data-miq_observe" => {:url => url}.to_json)
+                      options_for_select(dialog_types,
+                                          @edit[:new][:dialog_type]),
+                      :class => "selectpicker show-tick",                      
```

Comment:
@PanSpagetka i see you are using patternfly select picker here but it seems like it not working correctly, check my PR https://github.com/ManageIQ/manageiq/pull/3549 i had to make some JS calls to initialize the select box, look for miqInitSelectPicker JS function.